c++/Makefile.am

```diff
@@ -421,10 +421,14 @@ capnpc_c___SOURCES = src/capnp/compiler/capnpc-c++.c++
 # Also attempt to run ldconfig, because otherwise users get confused.  If
 # it fails (e.g. because the platform doesn't have it, or because the
 # user doesn't have root privileges), don't worry about it.
+#
+# We need to specify the path for OpenBSD.
 install-exec-hook:
 	ln -sf capnp $(DESTDIR)$(bindir)/capnpc
 	ldconfig < /dev/null > /dev/null 2>&1 || true
```

Comment:
Do you intentionally want to run `ldconfig` bare once, and then a second time with paths, on OpenBSD? Or should this be in an "else" block below?

Reply:
Technically an 'else' block is not required since the second `ldconfig` will overwrite the effect of the first `ldconfig`, but I see how an 'else' block can make things better. Updated.